miner/worker.go

```diff
@@ -1424,12 +1506,13 @@ func (w *worker) computeBundleGas(bundle types.MevBundle, parent *types.Block, h
 			// If tx is not in pending pool, count the gas fees
 			gasUsed := new(big.Int).SetUint64(receipt.GasUsed)
 			gasFees.Add(gasFees, gasUsed.Mul(gasUsed, tx.GasPrice()))
-
-			coinbaseBalanceAfter := state.GetBalance(w.coinbase)
-			coinbaseDelta := big.NewInt(0).Sub(coinbaseBalanceAfter, coinbaseBalanceBefore)
-			coinbaseDelta.Sub(coinbaseDelta, gasFees)
-			ethSentToCoinbase.Add(ethSentToCoinbase, coinbaseDelta)
 		}
+
+		coinbaseBalanceAfter := state.GetBalance(w.coinbase)
```

Comment:
This looks like a bad merge?

Reply:
you're right - from the proxy contract removal - will fix